src/Randomizer.SMZ3.Tracking/VoiceCommands/SpoilerModule.cs

```diff
@@ -829,5 +829,33 @@ private GrammarBuilder GetLocationUsefulnessHintRule()
 
             return GrammarBuilder.Combine(regionGrammar, roomGrammar);
         }
+
+        private string GetMultiworldSuffix(World locationWorld)
```

Comment:
Can we make this configurable? I'd prefer separate config entries for multiworld versions.

Reply:
Sure thing. I'll just take all of the current messages and create multiworld versions of them.